cmd/plasmacli/subcmd/eth/status.go

```diff
@@ -38,7 +39,7 @@ var statuscmd = &cobra.Command {
 		if synced {
 			num, err := client.LatestBlockNum()
 			if err != nil {
-				return fmt.Errorf("synced with eth node: %s \n error retrieving latest block height of eth endpoint: %s\n", conf.EthNodeURL , err)
+				return fmt.Errorf("eth node is syncing: %s", conf.EthNodeURL)
```

Comment:
rip, I meant for this change to be on line 46, the previous error message here was fine. The variable synced is either true or false, true if it is fully synced up or false if it still in catch up mode. Any crash would be reported by an earlier error. Also for future note, hit the "squash and merge" button when merging, it keep commit history more readable

Reply:
Whoops haha. Is there a best way to resolve? Will also squash & merge in the future

Reply:
open a new pr with the fix. Not sure if this has been done yet?